tests/unit/api/test_offline_ingest_query_graph.py

```diff
@@ -205,18 +208,18 @@ def odfv_feature_v231227_{feature_model.id}(
         inputs: pd.DataFrame,
     ) -> pd.DataFrame:
         df = pd.DataFrame()
+        feat = pd.to_datetime(inputs["__feature_V231227__part0"], utc=True)
```

Comment:
Curious what causes the changes in the generated views definition?

Reply:
Change in offline table name causes changes in node parameters. Change in node parameters causes change in node hash & it affects the sequence we used to traversal the decomposed graph. That's why the ordering of the statements is affected.